packages/backend/src/core/reports/ReportUpdater.ts

```diff
@@ -32,6 +30,7 @@ export class ReportUpdater {
     private readonly logger: Logger,
   ) {
     this.logger = this.logger.for(this)
+    // TODO(radomski): This config hash should be generated from only CBV projects
```

Comment:
change the name of the updater to CBVUpdater, and the hash function to the getCBVConfigHash